test/interop/C/multilocale.skipif

```diff
@@ -2,25 +2,44 @@
 
 """
  Multilocale interoperability requires ZMQ
- Installation of the ZMQ library is detected with the find_library function,
- which looks for the appropriate dynamic library (e.g. libzmq.so).
- Note that if the dynamic library is found, this test assumes that the
- header and static library are available. Can be overridden with
- `CHPL_TEST_NO_ZMQ=True`
+ Initial installation probe of the ZMQ library is detected with the
+ find_library function, but that only checks for libzmq.so and not the headers
+ so do a configure-like compilation check for the headers.
+
+ Longer term want to just do pkg-config check, but that requires updating MLI
+ support: https://github.com/Cray/chapel-private/issues/4292
 """
 
 from ctypes.util import find_library
 import os
+import shlex
+import subprocess
+import tempfile
 
-# Is ZMQ available?
+# Is ZMQ shared object available?
 zmq_found = find_library('zmq') is not None
-user_no_zmq = os.getenv('CHPL_TEST_NO_ZMQ') is not None
```

Comment:
I'm not seeing any other references to CHPL_TEST_NO_ZMQ in either this repo or our Jenkins set up.  Hopefully that means that no one is relying on it for their paratests either but it might be good to warn people to look for it.  Other than that, this looks good to me!

Reply:
It was originally added in https://github.com/chapel-lang/chapel/pull/20191 to support manual testing on SLES15 based EX machines. It's no longer needed since all the skipifs that used to use it now either use pkg-config or this configure-like test. I removed it from the EX testing "docs" in https://github.com/Cray/chapel-private/issues/3153 though.

Reply:
Thanks!